src/renderer/components/ConnectionsNew/ConnectionList.vue

```diff
@@ -1,6 +1,6 @@
 <template >
   <div v-if="list.length">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
For now, I think we should refrain from changing themes for consistency's sake. If we decide to move to a dark theme, this should be done everywhere at once instead of piecemeal. For now, please revert this change.